### ISSUE TITLE
Ability to instrument only permitted classes

### ISSUE THREAD

**Issue by LlamaLad7** [Kotlin/kotlinx-kover#673]
**What is your use-case and why do you need this feature?**
In the Kotlin compiler, there are a lot of tests which consider all the fields of a generated class, e.g. [here](https://github.com/JetBrains/kotlin/blob/96205cabfdb14a5aa5b1f0127871cee9c09aaef9/compiler/testData/codegen/box/closures/noRefToOuter.kt#L15), and because they use classfile version 52 (Java 8), the coverage agent adds a field to the class which breaks the test:
```
Fail: lambda should only capture 's': [final java.lang.String A$f$1.$s, private static transient boolean[] A$f$1.__$hits$__]
```
**Describe the solution you'd like**
Ideally I would be able to whitelist the specific compiler classes I am interested in instead of having to blacklist the offending test classes. 
Something like:
```kts
kover {
    currentProject {
        instrumentation {
            includedClasses.add("org.jetbrains.kotlin.fir.*")
        }
    }
}
```
to match the existing `excludedClasses` option.

**Comment by LlamaLad7** [comment]
Related: https://github.com/JetBrains/intellij-coverage/pull/64

**Comment by shanshin** [comment]
Resolved in `0.9.0-RC`
